core/server/services/labs.js

```diff
@@ -1,11 +1,16 @@
-var settingsCache = require('./settings/cache'),
-    _ = require('lodash'),
-    Promise = require('bluebird'),
-    SafeString = require('./themes/engine').SafeString,
-    common = require('../lib/common'),
-    labs = module.exports = {};
+const settingsCache = require('./settings/cache');
+const _ = require('lodash');
+const Promise = require('bluebird');
+const SafeString = require('./themes/engine').SafeString;
+const common = require('../lib/common');
+const config = require('../config');
+let labs = module.exports = {};
+
 
 labs.isSet = function isSet(flag) {
+    if (flag === 'members' && config.get('enableDeveloperExperiments')) {
```

Comment:
Is this gonna work for checking the labs flag in casper? e.g. https://github.com/TryGhost/Casper/blob/master/partials/site-nav.hbs#L23

Reply:
Not yet, have to check for that. 👍

Reply:
This should be an `||` right?